src/Angor/Client/Pages/Wallet.razor

```diff
@@ -1901,4 +1877,23 @@ else
         NavMenuState.SetActivePage("wallet");
         base.OnInitialized();
     }
+    
+    private long GetRecommendedMinFee()
+    {
+        // Return a recommended minimum fee rate based on network conditions
+        return FeeEstimations.Fees.Any() 
+            ? Math.Max(1, FeeEstimations.Fees.Min(f => f.FeeRate)) 
+            : 5;
+    }
+    
+    private async Task CustomFeeRateChanged(long customFeeRate)
```

Comment:
I think I would rather have one method that takes the fee and recalculates it, and then the changed event would call that method, and this can be the pattern everywhere else.
so basically what I am saying is the method `FeeRangeChanged` should call `CustomFeeRateChanged` however maybe we should rename it to `ReCalculateFee`